documentation/src/docs/pravega-write-read-methods.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Methods for Writing and Reading Pravega Streams
+---
+
+import { IfHaveFeature, IfMissingFeature, SDPVersion } from 'nautilus-docs';
+
+## Overview
+
+This table describes the methods that can be used to write and read data in Pravega streams. It is intended to be a comprehensive list which includes open-source, experimental, and commercial products that integrate with Pravega.
+
+Method                                                                                | Write | Transactional Write | Read | Special Features | Notes
+--------------------------------------------------------------------------------------|-------|---------------------|------|------------------|------------------------------------------------------
+[Pravega Sensor Collector](#pravega-sensor-collector)                                 | Yes   | Yes                 | No   | EO               |
+[SDP Ingest Gateway](#sdp-ingest-gateway)                                             | Yes   | No                  | No   |                  |
+[Pravega Ingest Gateway](https://github.com/pravega/pravega-ingest-gateway)           | Yes   | No                  | Yes  |                  |
+[Pravega Search](#pravega-search)                                                     | Yes   | Yes                 | Yes  | EO               |
+[Apache Flink](../flink-connectors/overview.md)                                       | Yes   | Yes                 | Yes  | EO               |
```

Comment:
We need to bring back submodule docs so these "../{flink,spark}-connectors/" links work...

Reply:
Can we get these docs to render from here: https://github.com/pravega/docs such that these links work?